main.cpp

```diff
@@ -41,10 +80,25 @@ struct Human
     float weight = 25.5f;
     float height = 160.5f;
 
-    void eat ();
+    void wakeUp();
     bool isHungry(int timeAwake, int stamina);
 };
 
+void Human::wakeUp()
+{
+    //call a waking up functionality
+}
+
+bool Human::isHungry(int timeAwake, int stamina)
+{
+    if (timeAwake > 8 && stamina <4 ) 
```

Comment:
add some space around your operators.  help the reader out.
- [ ] add space around `<` to improve readability